pkg/scheduler/core/assignment.go

```diff
@@ -49,6 +49,16 @@ const (
 	DynamicWeightStrategy = "DynamicWeight"
 )
 
+type scheduleMode string
+
+const (
+	// Lazy means that the last scheduling result will be referenced and maintained as much as possible.
+	Lazy scheduleMode = "Lazy"
+
+	// Shuffled means a complete recalculation without referring to the last scheduling results.
+	Shuffled scheduleMode = "Shuffled"
+)
+
```

Comment:
```suggestion
// assignmentMode indicates how to assign replicas, especially in case of re-assignment.
type assignmentMode string

const (
	// Steady represents a steady, incremental approach to re-assign replicas
	// across clusters. It aims to maintain the exist replicas distribution as
	// closely as possible, only making minimal adjustments when necessary.
	// It minimizes disruptions and preserves the balance across clusters.
	Steady assignmentMode = "Steady"

	// Fresh means that disregards the previous assignment entirely and
	// seeks to establish an entirely new replica distribution across clusters.
	// It is willing to accept significant changes even if it involves disruption.
	Fresh assignmentMode = "Fresh"
)
```